### PR TITLE
Mnoe dev toolkit

### PR DESCRIPTION
**Pull request by xaun**
@cesar-tonnoir please review - not ready for merge though. I need to update the developer docs, which I will do once @ouranos get's back to me about CORS so I can commit & document the uat urls in default settings. :beers:

### CODE REVIEW

**Comment by xaun** [comment]
screen shot 2016-10-20 at 18 19 40
screen shot 2016-10-20 at 18 21 58

File: workspace/app/services/dev-settings.svc.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+  this.defaults = function () {
+    return angular.copy(_self._defaults);
+  };
+
+  this.resetDefaults = function () {
+    return _self._defaults = angular.copy(DEFAULTS);
+  }
+
+  // NOTE: this will be refactored a lot when mnoe routes are added as default
```
Comment (cesar-tonnoir) [inline workspace/app/services/dev-settings.svc.js]: Indeed -> isn't it a good occasion to do it? :)
Looks like you've done 80% of the work below...

**Comment by xaun** [comment]
@ouranos please review the "Merge pull request #6 from xaun/remove-proxy" changes to see my working solution for providing CSRF support for the Dev Toolkit!

So you were right, Angular does not apply the CSRF token to CORS requests. Took a bit of work to figure out the best way to apply this, and once I did that, I realised that the CSRF token gets refreshed on various Angular Devise methods, so I had to come up with a little service to manage that. 

@cesar-tonnoir FYI & not ready for merge. 

Related PRs:
https://github.com/maestrano/impac/pull/379
https://github.com/maestrano/impac-express/pull/8

Remaining works:
- Deploy CORS configurations in the related PRs and test via https.
- Set the defaults urls to mnohub & impac uat.
- Refactor the impac-angular routes svc
- Update the developer.md & readme.md where needed

**Comment by ouranos** [comment]
Reviewed: https://github.com/xaun/impac-angular/pull/6
LGTM. Just need to edit the host to point to express isn't it?

**Comment by cesar-tonnoir** [comment]
@xaun is there anything preventing you from merging this PR?
If not can you please merge and update https://maestrano.atlassian.net/wiki/display/DEV/Temporary+workaround+for+the+Developer+Workspace (step 7.) once done ?

**Pull request merged**